samples/nextjs/package.json

```diff
@@ -11,6 +11,7 @@
     "sitecoreConfigPath": "/App_Config/Include/zzz",
```

Comment:
Something happened with dependencies here. Seems like order was changes, can you look at it? And leave dependencies which were added

Reply:
Could you please clarify which dependencies are not in the proper places? We used nothing but `npm install` to update this file - no manual changes were performed.